packages/core/test/components/chip.test.js

```diff
@@ -0,0 +1,62 @@
+import html from 'nanohtml';
+import Chip from '../../src/components/chip';
+
+function triggerEvent(el, type) {
```

Comment:
cant remember if this is used elsewhere but we maaaay want to abstract into a util, definitely not blocking on this